0006-cmx2cdp-protocol/README.md

```diff
@@ -1,84 +1,92 @@
 ---
-name: CMX to CDP protocol
-shortname: 6/CMX2CDP
-status: raw
+name: Configuration Data Transport protocol
+shortname: 6/CDT
```

Comment:
There is no confusion because of the letter and slash sign prefix in the protocol names. Which reminds me -- we need to adhere to that notation everywhere in the RFC text, so I'll fix this accordingly.